src/routes/common/address-info/address-info.helper.ts

```diff
@@ -35,8 +36,7 @@ export class AddressInfoHelper {
     address: string,
     source: Source = 'CONTRACT',
   ): Promise<AddressInfo | null> {
-    if (address == '0x0000000000000000000000000000000000000000')
-      return Promise.resolve(null);
+    if (address == NULL_ADDRESS) return Promise.resolve(null);
```

Comment:
After reviewing the usages of this fallback I think it is not worth it to have at this level (i.e. the `AddressInfoHelper` level).

The usages on the Safe Client Gateway of this fallback are only in Safe Info related logic: https://github.com/safe-global/safe-client-gateway/blob/eb481938a49c29b979fe98a5577f719f42a2b2eb/src/routes/safes/converters.rs#L49-L54

Therefore I would suggest removing this fallback. This can potentially be an issue to other endpoints that we have which should return the `NULL_ADDRESS` instead of `null`.

What do you think @hectorgomezv ?

Reply:
Pushed the suggested change here: https://github.com/safe-global/safe-client-gateway-nest/pull/380

This would then affect the changes on this PR which is more specific to Safe Info data.

Reply:
Yeah, I think it could be confusing, especially for a caller sending `NULL_ADDRESS` as you said, and it's not providing any benefit ATM. I'd vote for removing the fallback 👍

Reply:
Thanks for the thorough reviews and associated adjustments.

As the aforementioned change affects this PR, I will close this and we can open a separate one instead.